pandas_datareader/tests/yahoo/test_yahoo.py

```diff
@@ -131,6 +131,19 @@ def test_get_data_multiple_symbols(self):
         sl = ['AAPL', 'AMZN', 'GOOG']
         web.get_data_yahoo(sl, '2012')
 
+    @pytest.mark.parametrize('adj_pr', [True, False])
+    def test_get_data_null_as_missing_data(self, adj_pr):
+        # workaround as skip_on_exception decorator changes signature
+        # should it be changed to a signature-preserving decorator?
```

Comment:
yeah have to do something like this: https://stackoverflow.com/questions/12197263/pytest-parameterize-decorator-not-working-with-kwargs

@gfyoung if you want to fix up the skip_on_exception would be great (create an issue to start if you would)